wp-content/themes/bpr2018/resources/assets/sass/_changes.scss

```diff
@@ -0,0 +1,97 @@
+h2 {
```

Comment:
Try renaming this file to `_footer.scss` and put everything here in a `.footer { ... }` so that these styles don't potentially effect things outside of the footer.